.helm/ecamp3/templates/print_secrets.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "print.name" . }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+type: Opaque
+data:
+  {{- if .Values.print.sentryDsn }}
+  sentry-dsn: {{ .Values.print.sentryDsn | b64enc | quote }}
+  {{- end }}
```

```suggestion
  {{- else}}
  sentry-dsn: ""
  {{- end }}
```
Otherwise deployement fails, if no sentryDsn is provided explicitly: "Error: couldn't find key sentry-dsn in Secret default/ecamp3-print"